node/tests/dac.js

```diff
@@ -6,10 +6,10 @@ var val = 0;
 setInterval(function(){
 	console.log("dac =", val);
 	portB.pin[7].analogWrite(val);
-	
-	if (val < 3.2) {
-		val += 0.1;
+
+	if (val < 0.95) {
```

Comment:
To preserve a similar level of granularity to previous version. Used to write 33 values, now writes 30.